test/Microsoft.AspNetCore.Mvc.Core.Test/Filters/FilterCollectionTest.cs

```diff
@@ -42,16 +42,13 @@ public void Add_ThrowsOnNonIFilter()
             // Arrange
             var collection = new FilterCollection();
 
-            var expectedMessage =
-                $"The type '{typeof(NonFilter).FullName}' must derive from " +
-                $"'{typeof(IFilterMetadata).FullName}'." + Environment.NewLine +
-                "Parameter name: filterType";
+            var expectedMessage = $"The type '{typeof(NonFilter).FullName}' must derive from " + $"'{typeof(IFilterMetadata).FullName}'.";
 
             // Act & Assert
-            var ex = Assert.Throws<ArgumentException>(() => { collection.Add(typeof(NonFilter)); });
-
-            // Assert
-            Assert.Equal(expectedMessage, ex.Message);
+            ExceptionAssert.ThrowsArgument(
+                () => { collection.Add(typeof(NonFilter)); },
```

Comment:
No need to change this PR but FYI: `() => collection.Add(typeof(NonFilter)),` is all you need on this line.